operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2921,6 +2928,77 @@ admin_api:
   env: []
   extraEnvFrom: []
 
+# -- Cache for admin bucket.
+# If this is disabled, in-memory cache will be set by default.
+# You can use Redis too for cache and set the configuration via structuredConfig.
+# See GEM documentation for Redis configuration option.
+admin-cache:
+  # -- Specifies whether admin-cache using memcached should be enabled
+  enabled: false
```

Comment:
Curious - can this be always enabled? Does the feature require any specific migrations to be enabled or can we just enabled it for everyone upon upgrade?

FWIW the hardware cost should be negligible.

Reply:
No specific migration. But I just don't want users to get surprised to get another Memcached server suddenly.

@Logiraptor @56quarters what do you think?